src/scss/components/_progress-indicator.scss

```
@@ -84,8 +84,14 @@
 
 .#{prefix('progress-indicator-item')} {
   display: flex;
+  padding: 10px;
   justify-content: flex-start;
 
+  @include bem.modifier('not-applicable') {
+    display: none;
+    padding: 0;
+  }
```

I think that this is how you would specify this property with the design token, maintaining the current default behaviour:

```suggestion
  @include bem.modifier('not-applicable') {
    display: var(--of-progress-indicator-item-not-applicable-display, flex);
  }
```